src/pages/index.js

```diff
@@ -271,6 +271,7 @@ export default function HomePage() {
                 <p className="mt-5 max-w-prose mx-auto text-xl text-gray-500">
                   Algomart is a fully-baked solution that allows you to have a completely functional NFT marketplace as your jumping off point. Get your MVP launched today, and iterate as you experiment with NFTs and the value it can bring your customers.
                 </p>
+                <p className="mx-auto mt-5 text-xl text-gray-500 max-w-prose">You can try out the demo <a className="text-indigo-600" href="https://demo.algomart.dev/" target="_blank" rel="noopener noreferrer">here</a></p>
```

Comment:
Probably should add a disclaimer here (and ideally on the demo site too) that says it runs on TestNet and that you should only use [test card numbers](https://developers.circle.com/docs/test-card-numbers). And that the data may be wiped without notice. And that it might be down/break occasionally due to tests etc...

Reply:
Makes perfect sense! I'll start by adding in those notes and we can update as more come to mind

Reply:
Actually, should this be something we include on the demo site itself? Maybe some sort of banner or even a popup on first visit to which the user has to click 'okay'?

Reply:
@smonn how do you feel about pre-populating those card inputs on the demo (and possibly making them disabled too) when it's pointed to testnet? I don't _think_ that's an issue for the saved card feature with Circle but I could be mistaken.